router/pump.go

```diff
@@ -396,3 +402,12 @@ func (cp *containerPump) remove(logstream chan *Message) {
 	defer cp.Unlock()
 	delete(cp.logstreams, logstream)
 }
+
+func parseLogLine(line string) (string, time.Time, error) {
+	logEntry := strings.SplitN(strings.TrimSuffix(line, "\n"), " ", 2)
+	logTime, err := time.Parse(time.RFC3339Nano, logEntry[0])
+	if err != nil {
+		return "", time.Time{}, err
+	}
+	return logEntry[1], logTime, nil
```

Comment:
there is no guarantee that `logEntry[1]` exists. You should rework this function to check the length of `logEntry`